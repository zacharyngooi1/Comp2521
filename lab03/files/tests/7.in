50
10
45
15
40
20
35
25
30

