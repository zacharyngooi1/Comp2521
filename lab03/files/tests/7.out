Tree:
 50
 /
10
 \
 45
 /
15
 \
 40
 /
20
 \
 35
 /
25
 \
 30

#nodes = 9,  #leaves = 1
Infix  : 10 15 20 25 30 35 40 45 50 
Prefix : 50 10 45 15 40 20 35 25 30 
Postfix: 30 25 35 20 40 15 45 10 50 
ByLevel: 50 10 45 15 40 20 35 25 30 
